package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "espn-fantasy-autopick",
-    "version": "3.0.3",
-    "description": "A Chrome extension that lets you automatically add active players to the current roster in an ESPN NBA/NHL fantasy league.",
+    "version": "3.0.4",
```

Comment:
You can leave the version numbers as they are. I'll increment them in a separate commit for git tagging purposes.

Reply:
Ah cool, I will keep that in mind. Reverted back to `3.0.3`